ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-chain.yaml

```diff
@@ -5,6 +5,7 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-ibmcloud
   - ref: ipi-conf-ibmcloud-manual-creds
+  - ref: ipi-install-monitoringpvc
```

Comment:
Is this step a MUST, if not, pls not modify the existing one. If we want to add the step, we can add the step in provision chain.

Reply:
check the existed code, how about add to ci-operator/step-registry/ipi/conf/ibmcloud/sharednetwork/ipi-conf-ibmcloud-sharednetwork-chain.yaml